context/src/otelInBraveTest/java/io/opentelemetry/context/OtelInBraveTest.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.context;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import brave.Tracing;
+import brave.propagation.CurrentTraceContext;
+import brave.propagation.TraceContext;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.atomic.AtomicReference;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class OtelInBraveTest {
+
+  private static final ContextKey<String> ANIMAL = ContextKey.named("animal");
+  private static final Context CONTEXT_WITH_ANIMAL = Context.root().with(ANIMAL, "japan");
```

Comment:
an unusual animal. :)

Reply:
@anuraaga is guilty for this :)) and me that I copied paste